Source/WebCore/css/StyleRule.cpp

```diff
@@ -199,30 +199,48 @@ Ref<CSSRule> StyleRuleBase::createCSSOMWrapper(CSSStyleSheet* parentSheet, CSSGr
 
 unsigned StyleRule::averageSizeInBytes()
 {
-    return sizeof(StyleRule) + sizeof(CSSSelector) + StyleProperties::averageSizeInBytes();
+    return sizeof(StyleRule) + sizeof(CSSSelector) + StyleProperties::averageSizeInBytes() + +sizeof(WeakPtr<StyleRule>) + sizeof(Vector<Ref<StyleRuleBase>>);
```

Comment:
This looks wrong:

`StyleProperties::averageSizeInBytes() + +sizeof(WeakPtr<StyleRule>) + sizeof(`

I'm surprised it compiles!